packages/spatial/src/physics/systems/PhysicsPreTransformSystem.ts

```diff
@@ -168,6 +168,7 @@ const rigidbodyQuery = defineQuery([TransformComponent, RigidBodyComponent, Enti
 const colliderQuery = defineQuery([TransformComponent, ColliderComponent, EntityTreeComponent]) // @todo maybe add Not(RigidBodyComponent) to this query
 
 const filterAwakeCleanRigidbodies = (entity: Entity) => {
+  if (!RigidBodyComponent.initialized[entity]) return false
```

Comment:
aha looks like it was indeed a race condition with physics. i reckon rather than adding a new propery on the rigidbody component, we should check for the rigidbody in the physics world, which it kind of looks like `return !Physics.isSleeping(world, entity)` is kind of obscuring